src/utils/format-count.ts

```diff
@@ -10,7 +10,7 @@ export type NumberFormat =
 export type LabelFormat =
   | 'short' // [1.2]K
   | 'long'; // [1.2] thousand
-type Divisor = 1_000_000_000 | 1_000_000 | 1_000 | 1;
+export type Divisor = 1_000_000_000 | 1_000_000 | 1_000 | 1_0 | 1;
```

Comment:
Suggestion: I think it would be cleaner to leave this Divisor type the way it was before, and just use `number` as the type of `ndecimals` in the `formatUnitSize` function. (See also my other suggestion below)

Reasoning: The purpose of this type is conceptually related to units and language that only change every 3 orders of magnitude (K/M/G, thousand/million/billion). Including 10 as an option breaks that conceptual model.